elements/rh-skip-link/rh-skip-link.css

```diff
@@ -0,0 +1,62 @@
+:host {
+  display: block;
+  position: relative;
```

Comment:
This is fine but maybe we can put it on an inner container

Reply:
Moved these to `#rhds-skip-link-container`.